.gitignore

```diff
@@ -99,3 +99,6 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+# pyCharm
+*.idea
```

Comment:
This should be in a global git config on your machine instead of being checked in to the repo.